templates/docs/component-status.md

```diff
@@ -13,6 +13,28 @@ When we add, make significant updates, or deprecate a component we update their
 ### What's new in Vanilla {{versionMinor}}
 
 <table aria-label="What's new in Vanilla {{versionMinor}}">
+  <thead>
+    <tr>
+      <th style="width: 20%">Component</th>
+      <th style="width: 15%">Status</th>
+      <th style="width: 10%">Version</th>
+      <th style="width: 55%">Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- 2.31 -->
+    <tr>
+      <th><a href="/docs/patterns/navigation#dropdown">Navigation / Dropdown</a></th>
+      <td><div class="p-label--updated">Updated</div></td>
+      <td>2.31.0</td>
+      <td>We deprecated the <code>.p-subnav</code> class, which previously could theoretically be used outside of the main <code>.p-navigation</code> component. It has been succeeded by the <code>.p-navigation__item--dropdown-toggle</code> class.</td>
+    </tr>
```

Comment:
Please add the row about deprecated `p-subnav` as well.

It should be one row with deprecated `p-subnav` and one with new `p-navigation__item--dropdown-toggle`.